types/users.ts

```diff
@@ -9,7 +7,6 @@ export interface User {
   linkedinUrl?: string;
   personalSiteUrl?: string;
   selfIntro?: string;
```

Comment:
you should rebase/merge main first because of the type overhaul i pushed a few days ago